tsconfig.strict.json

```diff
@@ -387,11 +387,7 @@
     "lib/util/package-rules.ts",
     "lib/util/template/index.ts",
     "lib/versioning/api.ts",
-    "lib/versioning/aws-machine-image/index.ts",
```

Comment:
Bad merge?

Reply:
No, just marking other files ready in bulk

Reply:
Were they missed in their own PRs? I.e. already compliant?